.chloggen/add-opensearchexporter.yaml

```
@@ -0,0 +1,10 @@
+# Use this changelog template to create an entry for release notes.
+
+change_type: new_component
```

```suggestion
change_type: enhancement

```

Given that it was already mentioned as a new component in 0.82.0